onprc_ehr/src/org/labkey/onprc_ehr/notification/TreatmentAlertsFastsNotification.java

```diff
@@ -0,0 +1,65 @@
+package org.labkey.onprc_ehr.notification;
+
+import org.labkey.api.data.Container;
+import org.labkey.api.module.Module;
+import org.labkey.api.security.User;
+
+import java.util.Date;
+
+/**
+ * Created by kollil on 11/1/2021
+ */
+
+public class TreatmentAlertsFastsNotification extends ColonyAlertsNotification
+{
+    public TreatmentAlertsFastsNotification(Module owner)
+    {
+        super(owner);
+    }
+
+    @Override
+    public String getName()
+    {
+        return "Fasts treatment Notification";
+    }
+
+    @Override
+    public String getEmailSubject(Container c)
+    {
+        return "Fasts treatment Alerts: " + getDateTimeFormat(c).format(new Date());
+    }
+
+    @Override
+    public String getCronString()
+    {
+        return "0 0 5 * * ?";
+    }
+
+    @Override
+    public String getScheduleDescription()
+    {
+        return "every day at 6:30pm, 7Am and 7:30Am";
```

Comment:
This doesn't match the cron expression

Reply:
@labkey-jeckels , I have the timings right, but haven't worked on the cron setup yet. I wanted to test the query and data first, so, deployed the code into git.